src/content/docs/logs/get-started/live-archives.mdx

```diff
@@ -117,12 +97,16 @@ You can also edit an existing partition clicking the <Icon name="fe-more-horizon
    src={logsEditDataPartitionUsingLiveArchives}
 />
 
+For more information about how to use partitions, see our doc [Organize data with partitions](/docs/logs/ui-data/data-partitions/).
 </Step>
 
 <Step>
-### Query live archives with logs syntax [#query]
+## Query your logs
 
-There are two ways to query live archives: with Lucene or NRQL. If you want to use logs syntax that you're familiar with outside of New Relic, we suggest querying in the <DoNotTranslate>**Logs**</DoNotTranslate> UI. You'll need the **Live archives query** capability to query live archives.
+There are a few ways to query live archives: Lucene, NRQL, or the NerdGraph API. For all three methods, you'll need the **Live archives query** capability to query live archives.
+
+### Query live archives with logs syntax [#query]
+If you want to use logs syntax that you're familiar with outside of New Relic, we suggest querying in the <DoNotTranslate>**Logs**</DoNotTranslate> UI. 
```

Comment:
I would say that everyone who knows Lucene query syntax knows it's outside of New Relic. How about a revision like this: "If you're familiar with Lucene query syntax, you can execute it in our <DoNotTranslate>**Logs**</DoNotTranslate> UI.

Reply:
Also, "logs syntax" seems to general, which is why I'd suggest "Lucene query syntax."